processor/traceidprocessor/README.md

```diff
@@ -0,0 +1,32 @@
+# TraceID processor
```

Comment:
This processor has no real function outside of x-ray, I think we can add this logic into the x-ray exporter itself, defaulted off.

Reply:
Thanks. Will move this to the exporter with default off config.